clouddq/classes/entity_uri_schemes.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""todo: add classes docstring."""
+
+from enum import Enum
+from enum import unique
+
+
+@unique
+class EntityUriScheme(str, Enum):
+    """ """
+    DATAPLEX = "dataplex"
+    BIGQUERY = "bigquery"
```

Comment:
new classmethod called from_scheme(str) -> tries to create a new EntityUriScheme and returns it, catch Value error and throws NotImplementedError(err) instead